src/pages/TransactionDetails.vue

```diff
@@ -141,31 +161,28 @@ watch(
       </template>
     </BaseContentBlock>
     <BaseContentBlock
-      v-if="transaction"
       class="transaction-details__transactions"
+      :title="$t('transactions.executable')"
```

Comment:
Could you make it variable? e.g. depending on the executable, make it either "Instructions" or "WebAssembly Smartcontract"